module/applications/global/lp-history.mjs

```diff
@@ -0,0 +1,95 @@
+
+import LpTrackingData from "../../data/advancement/lp-tracking.mjs";
+
+/**
+ * 
+ * @param {object} inputs actor data
+ * @returns 
+ */
+export async function getLegendPointHistoryData( inputs ) {
```

Comment:
hmm, maybe change the name of the function to something more descriptive. This makes it sound like it actually returns some kind of data, which it doesn't.  So something more along the lines of "showLpHistoryPrompt".
Also, this should be a `static` function of the Class itself.